### PR TITLE
Task name in Gantt view made clickable

### PR DESCRIPTION
**Pull request by shreyashah115**
[#9117](https://github.com/frappe/erpnext/issues/9117)
screenshot from 2017-11-06 10-11-18

### CODE REVIEW

File: erpnext/projects/doctype/task/task_list.js [hunk 1]
```diff
@@ -25,7 +25,9 @@ frappe.listview_settings['Task'] = {
 		return [__(doc.status), colors[doc.status], "status,=," + doc.status];
 	},
 	gantt_custom_popup_html: function(ganttobj, task) {
-		var html = `<h5>${ganttobj.name}</h5>`;
+		var html = `<h5><a style="text-decoration:underline"\
```
Comment (netchampfaris) [inline erpnext/projects/doctype/task/task_list.js]: This will also work I think: `<a href="#Form/Task/${ganttobj.id}">`

**Review by netchampfaris — CHANGES_REQUESTED** [review]

File: erpnext/projects/doctype/task/task_list.js (hunk 1 above)
Comment (shreyashah115) [reply erpnext/projects/doctype/task/task_list.js]: Yep, it does. Updated!

**Comment by Priteshzeal** [comment]
[V11] it's not clickable to view / edit the Task.

ERPNext: v11.0.3-beta.34 () (staging)

Frappe Framework: v11.0.3-beta.48 () (staging)

**Comment by AdamN1** [comment]
Status for 2025.02, It's not working for the develop branch.
There will be new Gantt integration for ERPNext https://frappe.io/gantt I think it's solved there.

**Pull request merged**